website/src/components/SearchPage/SearchFullUI.tsx

```diff
@@ -84,6 +84,20 @@ export const InnerSearchFullUI = ({
 
     const [isColumnModalOpen, setIsColumnModalOpen] = useState(false);
 
+    const columnFieldItems: FieldItem[] = useMemo(
+        () =>
+            schema.metadata
+                .filter((field) => !(field.hideInSearchResultsTable ?? false))
+                .map((field) => ({
+                    name: field.name,
+                    displayName: field.displayName ?? field.name,
+                    header: field.header,
+                    alwaysSelected: field.name === schema.primaryKey,
```

Comment:
Weirdly AccessionVersion should be the primary key but it isn't marked as `alwaysSelected` on the Add Search Field section